pyproject.toml

```diff
@@ -56,3 +42,82 @@ module = [
   "pylsp_jsonrpc.*",
 ]
 ignore_missing_imports = true
+
+[tool.ruff]
```

Comment:
Is this change relevant for this PR? There's also the `ruff.toml` file which probably contains the same settings as mentioned here.

Reply:
Thanks, these were stupidly checked-in from when I was doing some testing of our JSON Schema...